controllers/deployment.go

```diff
@@ -121,11 +77,9 @@ func buildDeploymentPlanSpec(master *v1alpha1.CDAPMaster, labels map[string]stri
 		return &DeploymentPlanSpec{}, nil
 	}
 
-	// Get deployment plan depending on the number of pods desired
+	// Get the deployment plan. At the moment, it only supports numPods = 0 (i.e. one service per pod).
```

Comment:
Why zero means one? Should it better assign it to `1`?

Reply:
numPods here refers to the total number of pods used for deploying CDAP. 
numPods == 1--> all services in 1 pod 
numPods == 2--> UI service in 1 pod and backend services in 1 pod
...

numPods == 8 -> basically means each service run in its own pod, but instead of having user to cont the total number of services and set it to 8 to achieve one service per pod. Setting numPods = 0 or omitting it indicates one service per pod.